graph/edmondkarp.ts

```diff
@@ -0,0 +1,96 @@
+/**
+ * @description Compute the maximum flow from a source node to a sink node. The input graph is in adjacency list form. It is a multidimensional array of edges. graph[i] holds the edges for the i'th node. Each edge is a 2-tuple where the 0'th item is the destination node, and the 1'st item is the edge capacity.
+ * @Complexity_Analysis
+ * Time complexity: O(V * E^2) where V is the number of vertices and E is the number of edges
+ * Space Complexity: O(V^2) where V is the number of vertices
+ * @param {[number, number][][]} graph - The graph in adjacency list form
+ * @param {number} source - The source node
+ * @param {number} sink - The sink node
+ * @return {number} - The maximum flow from the source node to the sink node
+ * @see https://en.wikipedia.org/wiki/Edmonds%E2%80%93Karp_algorithm
+ */
+
+function edmondkarp(graph: [number, number][][], source: number, sink: number): number {
```

Comment:
This should be called `edmondsKarp`, and it needs an `export` (`default`). The file should also be called `edmond_karp.ts`.